### PR TITLE
Cal 206 update list of nitf attributes to documentation

### PR DESCRIPTION
**Pull request by ricklarsen**
#### What does this PR do?

Updates the list of NITF attributes in documentation to reflect behavior in v 0.2.0.

#### Who is reviewing it?

@peterhuffer @lcrosenbu @troymohl 
#### Choose 2 committers to review/merge the PR (please choose ONLY two committers from below, delete the rest).
@pklinef
@shaundmorris

#### How should this be tested?

verify list of attributes

#### What are the relevant tickets?

[CAL-206](https://codice.atlassian.net/browse/CAL-206)

#### Checklist:
- [x] Documentation Updated
	- [ ] Change Log Updated
- [ ] Update / Add Unit Tests
- [ ] Update / Add Integration Tests

### CODE REVIEW

**Comment by troymohl** [comment]
Should this contain the CsdidaAttribute as well?

**Comment by ricklarsen** [comment]
@troymohl Added.

File: distribution/docs/src/main/resources/_contents/_formats/supported-file-formats-contents.adoc [hunk 1]
```diff
@@ -0,0 +1,335 @@
...
+** `ext.nitf.image.image-declassification-exemption`
+** `ext.nitf.image.image-downgrade`
+** `ext.nitf.image.image-downgrade-date`
+** `ext.nitf.image.image-classification-text`
+** `ext.nitf.image.image-classification-authority-type`
+** `ext.nitf.image.image-classification-authority`
+** `ext.nitf.image.image-classification-reason`
+** `ext.nitf.image.image-security-source-date`
+** `ext.nitf.image.image-security-control-number`
+** `ext.nitf.image.imageSource`
```
Comment (lcrosenbu) [inline distribution/docs/src/main/resources/_contents/_formats/supported-file-formats-contents.adoc]: I believe this should be `image-source`

File: distribution/docs/src/main/resources/_contents/_formats/supported-file-formats-contents.adoc [hunk 2]
```diff
@@ -0,0 +1,335 @@
+
+====== NITF-specific Attributes Provided by ${cal-branding}
+
+${cal-branding} includes support for accessing additional metadata from specific file formats, primarily NITF. In addition to the above attributes, ${branding} can extract these attributes from NITF resources, if populated:
+
+.${cal-branding} Additional NITF Attributes
+
+${branding} has the capability to extract many NITF-specific Attributes from NITF files. (available if populated)
+
+* General Attributes
```
Comment (lcrosenbu) [inline distribution/docs/src/main/resources/_contents/_formats/supported-file-formats-contents.adoc]: A few General attributes I see missing.. were these purposely omitted?
- `title`
- `created`
- `modified`
- `effective`

File: distribution/docs/src/main/resources/_contents/_formats/supported-file-formats-contents.adoc [hunk 3]
```diff
@@ -0,0 +1,335 @@
...
+** `security.codewords`
+** `security.dissemination-controls`
+** `security.releasability`
+** `contact.creator-name`
+** `contact.creator-phone`
+** `isr.mission-id`
+** `isr.platform-id`
+** `isr.sensor-type`
+** `isr.sensor-id`
+** `isr.snow-cover`
```
Comment (lcrosenbu) [inline distribution/docs/src/main/resources/_contents/_formats/supported-file-formats-contents.adoc]: Missing `isr.niirs` normalized mapping in this list from the CSEXRA TRE.

File: distribution/docs/src/main/resources/_contents/_formats/supported-file-formats-contents.adoc (hunk 2 above)
Comment (ricklarsen) [inline distribution/docs/src/main/resources/_contents/_formats/supported-file-formats-contents.adoc]: They are included in the list of attributes common to all file types. As this is the second time I've gotten this comment, I'm going to include them in this list as well.

File: distribution/docs/src/main/resources/_contents/_formats/supported-file-formats-contents.adoc (hunk 3 above)
Comment (ricklarsen) [inline distribution/docs/src/main/resources/_contents/_formats/supported-file-formats-contents.adoc]: added.

File: distribution/docs/src/main/resources/_contents/_formats/supported-file-formats-contents.adoc (hunk 1 above)
Comment (ricklarsen) [inline distribution/docs/src/main/resources/_contents/_formats/supported-file-formats-contents.adoc]: updated.

**Pull request merged**